Example/android/app/build.gradle

```diff
@@ -125,7 +130,7 @@ android {
         variant.outputs.each { output ->
             // For each separate APK per architecture, set a unique version code as described here:
             // http://tools.android.com/tech-docs/new-build-system/user-guide/apk-splits
-            def versionCodes = ["armeabi-v7a":1, "x86":2]
+            def versionCodes = ["armeabi-v7a": 1, "x86": 2]
```

Comment:
unexpected changes no?

Reply:
Actually expected, usual autoformatter style for .gradle files.
Of course, If it would be a problem I will revert these changes.